test/ts/customRunner.ts

```diff
@@ -0,0 +1,39 @@
+import { resolve } from 'path'
+import runner, { RunnerOption } from '../../dist'
+
+const run = async () => {
+  try {
+    const options: Omit<RunnerOption, 'direction'> & { databaseUrl: string } = {
```

Comment:
&& ?

Reply:
No, it is joining of TS types
(and `& { databaseUrl: string }` is actually redundant, but TS complains without it...)